Strategy11/formidable-forms
# Add a condition to displaying name your form modal

Fix https://github.com/Strategy11/formidable-pro/issues/5036

We shouldn't show the form name popup when a form name is entered.
![CleanShot 2024-07-02 at 21 03 35](https://github.com/Strategy11/formidable-forms/assets/41271840/16014fca-5351-479f-a136-f87f614f715a)

### Test steps
1. Create a new form
2. Go to the form settings without saving the new form
3. Enter something in the form title input
4. Click the update button
5. Confirm that the form name modal does not show up in the process.
